Xamarin.Forms.Core/NavigationPage.cs

```diff
@@ -54,12 +56,14 @@ public NavigationPage(Page root) : this()
 			PushPage(root);
 		}
 
-		public Color BarBackgroundColor {
+		public Color BarBackgroundColor
```

Comment:
@StephaneDelcroix can we make this an attached property?

In more recent code it's been moved here
https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Core/BarElement.cs#L8